samples/simple_private_memory/simple_private_memory.cpp

```diff
@@ -33,14 +33,12 @@ using namespace cl::sycl;
  * hierarchical parallel_for_work_item context */
 static inline cl::sycl::id<1> get_global_id(cl::sycl::group<1>& group,
```

Comment:
Oh yes, I remember and Victor worked on this. I put this into triSYCL long time ago because it was obvious to me it was forgotten from the spec... :-(